projects/cdk/schematics/ng-update/v3-55/index.ts

```diff
@@ -0,0 +1,37 @@
+import {Rule, SchematicContext, Tree} from '@angular-devkit/schematics';
+import {NodePackageInstallTask} from '@angular-devkit/schematics/tasks';
+import {
+    addPackageJsonDependency,
+    getPackageJsonDependency,
+    saveActiveProject,
+} from 'ng-morph';
+
+import {TAIGA_VERSION} from '../../ng-add/constants/versions';
+import {TuiSchema} from '../../ng-add/schema';
+import {FINISH_SYMBOL, titleLog} from '../../utils/colored-log';
+import {getFileSystem} from '../utils/get-file-system';
+
+export function updateToV3_55(options: TuiSchema): Rule {
+    return (tree: Tree, context: SchematicContext) => {
+        if (!getPackageJsonDependency(tree, `@taiga-ui/proprietary-core`)) {
```

Comment:
`@taiga-ui/experimental` is peer dependency of `@taiga-ui/proprietary-core`.
It should be install automatically by modern npm (7+) (if `--legacy-peer-deps` flag is not enabled).

Do we actually need to install this package manually ?